src/ipa_pc/data_structures.rs

```diff
@@ -136,255 +198,18 @@ impl<G: AffineCurve> PCVerifierKey for VerifierKey<G> {
     }
 
     /// Randomize key for testing purpose
-    fn randomize(&mut self) {
+    // #[cfg(test)]
```

Comment:
Same comment as other ones

Reply:
Answered above

Reply:
Either you move these changes here and report them on ginger-lib monorepo branch,
or we formally decide to address this changes in ginger-lib, but you need to create a separate branch from the monorepo branch containing them.
Currently, it's difficult to review them as in the monorepo branch everything will appear as new.

Reply:
Done